.circleci/config.yml

```diff
@@ -295,44 +291,31 @@ commands:
             ng build $ANGULAR_PROJECT_NAME $BUILD_PARAMS --configuration production --aot --base-href=/ --output-path=$OUTPUT_DIR --verbose=true --progress=true
           #
 
-  conditionally-launch-build-and-deploy:
-    description: Check every study, launch build and deploy if study has changed
+  halt-when-study-unchanged:
```

Comment:
`app-build`, `app-deploy` and `ui-unit-test` jobs will have this optional check to determine whether job should continue running or stopping if study has not changed.